features/clipboard-custom-format.yml

```
@@ -0,0 +1,6 @@
+name: Async clipboard support for custom formats
+description: Support for custom formats in the `navigator.clipboard` API allows you to read and write unsanitized custom data in the system clipboard. Custom formats have mime-types that start with the `web ` prefix.
```

"Custom formats" and the `web ` prefix seem like the most important information here. I'm suggesting front-loading this information. I also think we can be more specific about which interface this actually applies to, which should help relate this information to documentation.

(Also I feel like MIME types are a bit old school but I could be convinced otherwise)

```suggestion
name: Custom formats for clipboard items
description: The `web ` prefix for `ClipboardItem` data types (also known as MIME types) allows reading and writing ununsanitized custom data from the device clipboard.
```